loader/standard_loader.py

```diff
@@ -1,11 +1,135 @@
+import logging
+import pprint
+import re
+import typing
+from collections import namedtuple
+from typing import Set
+
 from loader.base_loader import DssUploader, MetadataFileUploader
 
+logger = logging.getLogger(__name__)
+
+SCHEMA_URL = ('https://raw.githubusercontent.com/DataBiosphere/metadata-schema/master/'
+              'json_schema/cgp/gen3/2.0.0/cgp_gen3_metadata.json')
+
+
+class ParsedBundle(namedtuple('ParsedBundle', ['bundle_uuid', 'metadata_dict', 'data_objects'])):
+
+    def pprint(self):
+        return pprint.pformat(self, indent=4)
 
 class StandardFormatBundleUploader:
     def __init__(self, dss_uploader: DssUploader, metadata_file_uploader: MetadataFileUploader) -> None:
         self.dss_uploader = dss_uploader
         self.metadata_file_uploader = metadata_file_uploader
 
-    def load_all_bundles(self, input_json: dict):
-        # TODO Implement this loader
-        raise NotImplemented
+    @staticmethod
+    def _parse_bundle(bundle: dict) -> ParsedBundle:
+        try:
+            data_bundle = bundle['data_bundle']
+            bundle_uuid = data_bundle['id']
+            metadata_dict = data_bundle['user_metadata']
+            data_objects = bundle['data_objects']
+        except KeyError:
+            # TODO: Make a bundle parse error that spits back whatever info we can get about the bundle
+            raise
+        return ParsedBundle(bundle_uuid, metadata_dict, data_objects)
+
+    @staticmethod
+    def _get_file_uuid(file_guid: str):
+        uuid_regex = re.compile('[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}')
```

Comment:
This line could be moved to the class level, so avoid unnecessary recompilation of the regex each time this method is called.

Reply:
I'm going to ignore this for now since:
>Note The compiled versions of the most recent patterns passed to re.compile() and the module-level matching functions are cached, so programs that use only a few regular expressions at a time needn’t worry about compiling regular expressions.

https://docs.python.org/3/library/re.html#re.compile

Reply:
Never mind, I just fixed it anyway.